### DISCUSSION TITLE
Error:Undefined property: PowerComponents\LivewirePowerGrid\Button::$param

### DISCUSSION THREAD

**Discussion by koronaldo5863** [Power-Components/livewire-powergrid#707] [BUG]
Hello.

In version :3.2.8  this code works:

    public function header(): array
    {
        
        return [
            Button::add('new-modal')
            ->caption('Add bank')
            ->class('btn btn-primary btn-sm')
            ->openModal('bank-edit-modal', [
                'bankId' => '0',
                
            ]),
        ];
    }

Now in  version 3.5.10 not works.

this error appears:  Undefined property: PowerComponents\LivewirePowerGrid\Button::$param

**Comment by luanfreitasdev** [comment]
Can you post more details of the error?

**Reply by koronaldo5863** [reply]
" works in version 3.2.8 powergrid, after upgrade for 3.5.10 not works.

When I load grid in my blade, The erros is caused:

![image](https://user-images.githubusercontent.com/51420905/200385222-96664e02-049e-45b2-a95e-0ee5951dbb60.png)

**Reply by koronaldo5863** [reply]
The error occurred in upgrade version 3.3.0 for 3.4.0

**Comment by luanfreitasdev** [comment]
What happened is that between one version and another the name of the variable changed to "params". In this case, you must update the views in your environment

**Reply by koronaldo5863** [reply]
Solved.

https://github.com/Power-Components/livewire-powergrid/commit/6e52f7682bcd1d6e8a8b168e4a05f0c86505f577#diff-b7147be6e742b30438f3596646eca314119bf2eca5036b2fb4bd8d249da90ed1
